renderer/HardwareOpenGL.cpp

```diff
@@ -754,6 +754,7 @@ void opengl_Close(const bool just_resizing) {
     OpenGL_cache_initted = 0;
   }
 
+  SDL_GL_UnloadLibrary();
```

Comment:
This change leads to an exception that `_this->gl_data` is a `nullptr` in SDL_windowsopengl.c Line 752 when exiting any mission by pressing <ESC> and hitting <YES>.

callstack:
```
Descent3.exe!WIN_GL_CreateContext(SDL_VideoDevice * _this, SDL_Window * window) Line 752
Descent3.exe!SDL_GL_CreateContext_REAL(SDL_Window * window) Line 4066
Descent3.exe!SDL_GL_CreateContext(SDL_Window * a) Line 590
Descent3.exe!opengl_Setup(oeApplication * app, const int * width, const int * height) Line 438
Descent3.exe!opengl_Init(oeApplication * app, renderer_preferred_state * pref_state) Line 558
Descent3.exe!rend_ReInit() Line 1741
Descent3.exe!rend_SetPreferredState(renderer_preferred_state * pref_state) Line 375
Descent3.exe!SetScreenMode(int sm, bool force_res_change) Line 997
Descent3.exe!MainMenu() Line 733
Descent3.exe!MainLoop() Line 545
Descent3.exe!Descent3() Line 507
Descent3.exe!oeD3LnxApp::run() Line 142
Descent3.exe!WinMain(HINSTANCE__ * hInst, HINSTANCE__ * hPrevInst, char * szCmdLine, int nCmdShow) Line 324

```

Reply:
On closer investigation, this might be related to running the game with `-width 1920 -height 1080`.

Reply:
Probably better move SDL_GL_UnloadLibrary() to rend_Close() then.

Reply:
Confirmed to work now.